assignments/array-methods.js

```diff
@@ -54,30 +57,68 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.
+// Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach( (runner) => {
```

Comment:
great way to do this. remember that you can omit the parens if your function only has one parameter, and you can omit the curly braces if you are only returning one line.

```
runners.forEach(runner =>
  fullName.push(`${runner.first_name} ${runner.last_name}`)
);
```